components/Dates/ShowDatesList/HandelClickDate.js

```diff
@@ -0,0 +1,3 @@
+export default function handelClickDate({ index, setListEntry }) {
+  setListEntry(index);
+}
```

Comment:
I don't know if this should be a function in an external file. You can call the `setListEntry` function directly without any indirection.